hack/jenkins/common.sh

```diff
@@ -360,11 +360,21 @@ then
     EXTRA_START_ARGS="${EXTRA_START_ARGS} --container-runtime=${CONTAINER_RUNTIME}"
 fi
 
+if test -f "${JSON_OUT}"; then
+  rm "${JSON_OUT}" || true # clean up previous runs of same build
+fi
+
+touch "${JSON_OUT}"
+
 ${SUDO_PREFIX}${E2E_BIN} \
   -minikube-start-args="--driver=${DRIVER} ${EXTRA_START_ARGS}" \
   -test.timeout=${TIMEOUT} -test.v \
   ${EXTRA_TEST_ARGS} \
-  -binary="${MINIKUBE_BIN}" 2>&1 | tee "${TEST_OUT}"
+  -binary="${MINIKUBE_BIN}" 2>&1\
+  | go tool test2json -t\
+  | gotestsum --jsonfile "${JSON_OUT}" -f standard-verbose\
```

Comment:
this is not installed, u would wanna install it before running it.

Reply:
Now we are installing. Windows also just downloads a specific binary from their releases and untars it.